lib/components/table/table.test.ts

```diff
@@ -0,0 +1,366 @@
+                        ${getTh("Month", "month", useButton)}
+                        ${getTh("Temperature", "temp", useButton)}
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>Winter</td>
+                        <td data-s-table-sort-val="12">December</td>
+                        <td>2</td>
+                    </tr>
+                    <tr>
+                        <td>Spring</td>
+                        <td data-s-table-sort-val="3">March</td>
+                        <td>13</td>
+                    </tr>
+                    <tr>
+                        <td>Summer</td>
+                        <td data-s-table-sort-val="6">June</td>
+                        <td>25</td>
+                    </tr>
+                    <tr>
+                        <td>Fall</td>
+                        <td data-s-table-sort-val="9">September</td>
+                        <td>13</td>
+                    </tr>
+                    <tr data-s-table-sort-to="bottom" class="fw-bold">
+                        <td colspan="2">Average temperature</td>
+                        <td>13</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    `;
+};
+
+describe("s-table", () => {
+    // Current, legacy markup
+    [true, false].forEach((useButton) => {
+        describe(`Column sorting indicators${
+            useButton ? "" : " (legacy markup)"
+        }`, () => {
+            const allColIds = [
+                "test-sortable-season-col",
+                "test-sortable-month-col",
+                "test-sortable-temp-col",
+            ] as const;
+            type ColId = AsLiterals<typeof allColIds>;
+
+            const allSvgIds = [
+                "test-sortable-season-svg-asc",
+                "test-sortable-season-svg-desc",
+                "test-sortable-season-svg-none",
+                "test-sortable-month-svg-asc",
+                "test-sortable-month-svg-desc",
+                "test-sortable-month-svg-none",
+                "test-sortable-temp-svg-asc",
+                "test-sortable-temp-svg-desc",
+                "test-sortable-temp-svg-none",
+            ] as const;
+            type SvgId = AsLiterals<typeof allSvgIds>;
+
+            const expectThisColumnToBeSorted = (
+                columnId?: ColId,
+                order?: SortOrder
+            ) => {
+                allColIds.forEach((colId) => {
+                    const columnEl = screen.getByTestId(colId);
+
+                    if (columnId === colId) {
+                        expect(columnEl).to.have.attribute("aria-sort", order);
+                    } else {
+                        expect(columnEl).to.not.have.attribute("aria-sort");
+                    }
+                });
+            };
+
+            const expectTheseIndicatorsToBeVisible = (visibleIds: SvgId[]) => {
```

Comment:
Small Rant, no action required:
My personal feeling is that we test too much implementation details here. Ideally we should test things from a user perspective, reaching out to element via accessible queries (e.g. `getByRole('button', { name: "Season"  })`) and then check for the presence of the aria-sort attribute. 
Testing the SVG visibility is a bit border line but I see why we would do that - we could mess it up.